### ISSUE TITLE
An early version of the playbook - some good content to merge in

### ISSUE THREAD

**Issue by iteles** [foundersandcoders/master-reference#42]
As per https://github.com/foundersandcoders/master-reference/issues/23#issuecomment-268468114 , we should merge these two repos together of course!

<hr/>
Original text:

I had *entirely* forgotten [this early version of a `master-reference`](https://github.com/foundersandcoders/playbook) until @njsfield pointed it out (thanks!)! 🎩 

As you can see, it never gained much traction, [with the same repeat offenders](https://github.com/foundersandcoders/playbook/graphs/contributors) contributing, _but_ apart from the fact that it's a historical treasure trove, there is some content _very_ worth putting into `master-reference`.

A thorough read-through is needed, but for a start (and all of these may well require updating):
+ [Code reviews](https://github.com/foundersandcoders/playbook/blob/916296bba49d54dbc7e939125656a9eb52182167/code-reviews.md) - clearly a @nelsonic piece of art
+ [House rules](https://github.com/foundersandcoders/playbook/blob/916296bba49d54dbc7e939125656a9eb52182167/house-rules.md) - still only a start but should exist here too, with provisos for both physical and virtual spaces, including reference to code of conduct
+ [Pair programming](https://github.com/foundersandcoders/playbook/blob/master/pair-programming.md)
+ Projects - the `md` file here doesn't say much but the key is we should have a section here on the processes we encourage students to use when working with clients too
+ [On teams](https://github.com/foundersandcoders/playbook/blob/master/teams.md) - some great content here - we need to find sources and use this as the scientific basis for our team selection/ways of working

**Comment by nelsonic** [comment]
@iteles is there a _reason_ we don't just _use_ the **PlayBook** repo instead of _duplicating_ the effort? was there something _wrong_ with it...?

**Comment by iteles** [comment]
Honestly, I'd completely forgotten it existed. I assume everyone else did too. A lot more time has gone into this one now so it would be worth lifting your pair programming and code review md's as a quick PR if you can spare a few minutes to get the commit credit for your work there

**Comment by nelsonic** [comment]
_Noooooo_ ... if things get "lifted" by _copy_-pasted ***again*** then we have ***learned nothing***. 😢 

In _every_ learning environment, copying without attribution [_which is trivial with a simple git merge_](http://stackoverflow.com/questions/1425892/how-do-you-merge-two-git-repositories) is frowned upon and in many it's a [disciplinary offence](http://www.plagiarism.org/plagiarism-101/what-is-plagiarism) 

I **100% believe** in `#OpenSource` but "lifting" without a second thought is _totally_ the _wrong_ way!

> I don't _care_ about "_credit_", I care about _wasted_ effort!

# "_It is amazing what you can accomplish if you do not care who gets the credit_" ~ Truman

I _care_ that (_for example_) the [code-reviews.md](https://github.com/foundersandcoders/playbook/blob/master/code-reviews.md) has been _ignored_ for the past **3 cohorts** and people have been _freestyling_ it instead of iterating on a step-by-step guide (_written by someone who has ***invested Years*** learning the craft in **many** organisations and from **hundreds** of books_) 😧 

> _Please `never` **compare** the amount of effort that has gone into "master reference" with the work that came before ... we all **know** @bradreeder, @des-des & others have done a **superb** job with planning the curriculum for F&C10 here, but "**forgetting**" previous work is **theft** of time!_

# _How_? did this _happen_...?

**Comment by iteles** [comment]
I couldn't agree more with you @nelsonic. I've updated the original comment and issue name to reflect we should merge the two together and left the original text just because there'/ already some work there on which md files we should keep and which are probably out of date and can be deprecated when we merge the two repos together.

**Comment by nelsonic** [comment]
Wow... if _ever_ there was a _clear_ sign that the time of the _most experienced_ people in the organisation has been _wasted_ this would be it. I'm _speechless_. 🙊  😡

**Comment by iteles** [comment]
This is *entirely* my fault. I *completely* forgot the playbook existed. I think that when it stopped being used I became frustrated and put it out of my mind. So when we started talking about 'master-reference', I didn't remember it. As I've mentioned above. 
Whoa. No more writing issues for me on little sleep and late at night!

**Comment by bradreeder** [comment]
Sorry @nelsonic I wasn't around at the time this repo was being used and wasn't aware it existed. Very happy for the two to be merged, or to move work from here across over the holidays, as it's essentially the point of this resource to build upon prior materials. Whatever is easier.

**Comment by nelsonic** [comment]
@bradreeder hope your day at work is going well. 👍 

I _understand_ that if people are not _encouraged_ to _look_ at the previous repos in the organisation then _nobody_ will _ever_ be aware of what has been done `before`...
it's not _your_ fault. and you are _doing_ something about making it better for future members.

# tl;dr ... 

> The following is not directed at any _one_ person.
it is merely my vain/desperate scream into the `int main(void) {...} `
if nobody voices their _deep_ disappointment at this _charade_, 
_nothing_ will change/improve, and all knowledge/learning will be _lost_!

# `<rant class='OMG'>`

# `<h1>` _`W`hat_ `T`he _`F`reak_ is going _on_!? `</h1>`

It _boggles_ my mind that with only ***4 pages*** of repos in https://github.com/foundersandcoders org 
![four-pages](https://cloud.githubusercontent.com/assets/194400/21389968/480da7e0-c77c-11e6-886d-0526e2d9bb2a.png)

_nobody_ `else` (_besides @njsfield_) has _bothered_ to read through them ... 😖 
no _wonder_ people were "_confused_" ... 😢 

> I'm _guessing_ nobody _explained_ to you guys that the **_single_ best way to improve your coding skills** is to **_read_ other people's `code`**: https://changelog.com/posts/one-sure-fire-way-to-improve-your-coding ❓ 

Hats off to @njsfield for making it _all-the-way_ to [**page 3**](https://github.com/foundersandcoders?page=3) (_oh the [**irony**](https://en.wikipedia.org/wiki/No_More_Page_3)...!_)  

![playbook-page-three](https://cloud.githubusercontent.com/assets/194400/21390208/9317bdba-c77d-11e6-8e66-74368685aeda.png)
Well done, _again_, Nick, for _re-surfacing_ the "**playbook**" that @sofer _painstakingly_ put together 
(_and a few of us contributed to..._)

I know that F&C is a _voluntary_ organisation where people contribute out of _good will_, 
but if a cohort cannot make the _effort_ to be ***proactive*** and read through **4 pages** of repos 
the sustainability is _questionable_ ... 😞 

![image](https://cloud.githubusercontent.com/assets/194400/21393649/4883c7f2-c78d-11e6-9e92-954ec286a062.png)

How can we be _expanding_ to a "_New School_" when London is not running like _clockwork_?
Are we _exporting_ the _Chaos_? or are we _relying_ on @des-des to "_save the day_"...?

# `</rant>`

**Comment by des-des** [comment]
good catch @njsfield !

So me and @bradreeder's focus has been on building a curriculum, and more importantly we are working to build _process_ around iterating it!! (we can see that was missing in the case of the playbook!).

Time wasted so far: 0 hours!

Me and brad have talked about how we wanted to build many of the documents in the playbook in the new year so this is perfect timing + wonderful!

### proposal going forward
 1. Merge this in to the current repo under the directory `old_playbook` (with git),
 2. We can then pull stuff out (`git mv`) / refactor it, with each file in its own pull request, so that we can get feedback!

I'd be happy to do this over the break. Going forward, @nelsonic do not despair

**Comment by bradreeder** [comment]
@nelsonic I find it disappointing as well. But to me dissatisfaction is one of the core motivators for innovation -- as a solution to a problem that pains us.

Anyone could propose any number of changes to avoid scenarios like these, but if they met resistance then it would be pointless. The important thing to me is that everyone is now on board with the changes that need to happen, there are enough capable people around with the time to implement those changes, and enough lessons like this for everyone to reach a consensus on _why_ past methods must be abandoned.

I think it's important that dissatisfaction is voiced and visible here, so people understand _why_ the changes we're trying to implement here are necessary. But it's also important that people reading this not become disheartened or demotivated and understand that they're here at a pivotal moment. We can genuinely change things right now. The green light is there.

![its-happening](https://media.giphy.com/media/rl0FOxdz7CcxO/giphy.gif)

I want to be an agent of change, and I welcome anyone who feels the same way to take on responsibilities and accelerate the process.

**Comment by sofer** [comment]
One reasonable option is to reference content in the playbook from the master-reference, instead of copying it, in the same way that we might want to link to the many excellent READMEs [elsewhere](https://github.com/dwyl?q=learn).

**Comment by iteles** [comment]
@sofer I think we've established above we'll merge the two repos 👍

**Comment by njsfield** [comment]
@des-des how will the merge be performed? Will it also merge in commit history from the playbook repo? Just curious as have never done it before!

**Comment by iteles** [comment]
Closer by mistake on GitHub mobile view :unimpressed: 
@njsfield http://stackoverflow.com/questions/1425892/how-do-you-merge-two-git-repositories

**Comment by des-des** [comment]
@njsfield I used this: http://stackoverflow.com/questions/1365541/how-to-move-files-from-one-git-repo-to-another-not-a-clone-preserving-history

Which has also worked for me in the past

**Comment by bradreeder** [comment]
Playbook content merged #45 

Some of these files are potentially outdated in terms of how the curriculum has been run in recent iterations. Others will need updating before the start of the next curriculum.

Shall I leave this issue open to discuss these @iteles or would you prefer me to close this and open a new issue?

**Comment by iteles** [comment]
Let's close and create new, more specific issues with a help wanted label to make it easier for anyone who has some time to contribute! :)

**Comment by bradreeder** [comment]
Gotcha. Closed #45
